packages/loader/container-loader/src/container.ts

```diff
@@ -424,7 +427,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             {
                 clientType, // Differentiating summarizer container from main container
                 loaderVersion: pkgVersion,
-                containerId: uuid(),
+                containerSessionId: this.containerSessionId,
```

Comment:
mixed on if i should change the telemetry or not. previously i used containerId everywhere eventhough i didn't really like it, now that i am using containerSessionId i like this being consistent, and it think the usage is clearer, but it could cause issue if anyone has a dependency on this for alerting/monitoring

Reply:
i'd be on board for not changing it because of how confusing/difficult it could be for someone depending on this field to find what changed.  the actual meaning of the field is still the same too

---
In reply to: [488988860](https://github.com/microsoft/FluidFramework/pull/3639#discussion_r488988860) [](ancestors = 488988860)